### PR TITLE
Deleting the XPlatCppSdk template

### PR DESCRIPTION
**Pull request by pgilmorepf**
... and replacing refs to XPlatCoreTemplate

### CODE REVIEW

**Comment by Vicodex** [comment]
Please don't commit this until after successful publish, and the scrum later today

File: SDKBuildScripts/Linux/XPlat_build.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SdkName="XPlatCppSdk"
-targetSrc="xplatcppsdk"
+targetSrc="XPlatCoreTemplate"
```
Comment (Vicodex) [inline SDKBuildScripts/Linux/XPlat_build.sh]: >XPlatCoreTemplate [](start = 11, length = 17)

Shouldn't this be all small case?

File: SDKBuildScripts/Windows/XPlat_build.bat [hunk 2]
```diff
@@ -1,6 +1,6 @@
 setlocal
 set SdkName=XPlatCppSdk
-set targetSrc=xplatcppsdk
+set targetSrc=XPlatCoreTemplate
```
Comment (Vicodex) [inline SDKBuildScripts/Windows/XPlat_build.bat]: >XPlatCoreTemplate [](start = 14, length = 17)

Shouldn't this be all small case?

**Comment by toddbellMSFT** [comment]
retest

**Comment by toddbellMSFT** [comment]
retest

**Comment by toddbellMSFT** [comment]
retest 

The generic sdk build machine had windows open, resulting in the failed tests.

**Comment by toddbellMSFT** [comment]
retest

**Pull request merged**